app/src/main/java/com/eveningoutpost/dexdrip/Models/blueReader.java

```diff
@@ -122,26 +160,39 @@ private static synchronized void processNewTransmitterData(TransmitterData trans
             PersistentStore.setLong("blueReader_Full_Battery", transmitterData.sensor_battery_level);
             Log.i(TAG, "blueReader_Full_Battery set to: " + transmitterData.sensor_battery_level) ;
         }
-        Pref.setInt("bridge_battery", ((transmitterData.sensor_battery_level - 3300) * 100 / (((int) (long) PersistentStore.getLong("blueReader_Full_Battery"))-3300)));
-        sensor.latest_battery_level = ((transmitterData.sensor_battery_level - 3300) * 100 / (((int) (long) PersistentStore.getLong("blueReader_Full_Battery"))-3300));
-        blueReaderDays = 6.129200670865791d / (1d + Math.pow(((double)transmitterData.sensor_battery_level/3763.700630306379d),(-61.04241888028577d)));
+        int localBridgeBattery =((transmitterData.sensor_battery_level - 3300) * 100 / (((int) (long) PersistentStore.getLong("blueReader_Full_Battery"))-3300));
```

Comment:
in your next pull-request you might want to extract a constant for 3300 with a name like "MIN_BATTERY_LEVEL"

Reply:
@ahaarrestad, yes, May in future. The problem is, that the max and min Battery seems to be felxible (depending on the version of the bluereader and the quality of the Battery). So after this pull request I want to collect raw batterydata from other users to make a better view on that, and maybe to make a flexible approach on the bat level...

@jamorham, not sure after the lot discussions and commits in that pull if you recognize that it looks like ok to merge - surely after you have a look on it. If you feel it is to complex due to the additive commits I will make a new one if you want to have it